src/testUtils.ts

```diff
@@ -340,7 +340,7 @@ export function cancelRunningTests(): Thenable<boolean> {
 function expandFilePathInOutput(output: string, cwd: string): string {
 	let lines = output.split('\n');
 	for (let i = 0; i < lines.length; i++) {
-		let matches = lines[i].match(/^\s*(.+.go):(\d+):/);
+		let matches = lines[i].match(/^\s*(\s*\w+.go):(\d+):/);
```

Comment:
Why do we want to capture the spaces here? Is this to cater to the case when the file name contains a space?

Reply:
I thought spaces might be present at the beginning of the file. This has another bug, where if we have spaces in the filename like `file name.go` the regex will fail to pick this. Another commit inc.